selftests/unit/test_utils_filelock.py

```diff
@@ -0,0 +1,45 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+from avocado.utils import filelock
+
+
+class TestAsset(unittest.TestCase):
```

Comment:
IMO it makes sense to name this `TestFileLock` instead... there are no asset references here.